{{cookiecutter.project_slug}}/README.md

```diff
@@ -3,6 +3,7 @@
 [![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit)
 [![Tests status][tests-badge]][tests-link]
 [![Linting status][linting-badge]][linting-link]
+[![Documentation status][documentation-badge]][linting-link]
```

Comment:
```suggestion
[![Documentation status][documentation-badge]][documentation-link]
```
But also needs guarding with `if deploy_docs_to_github_pages` doesn't it?

Reply:
Thanks for catching! With regards to putting in a conditional block - the `deploy_docs_to_github_pages` variable only controls if the documentation Actions workflow has an additional step which commits the built HTML documentation to a branch `gh-pages` on pushes to `main`, but even if this option is not selected the workflow will still test building the docs in strict mode on each pull-request / push to `main`.